packages/kit/src/components/OneKeyAuth/supabase/README.md

```
@@ -0,0 +1,177 @@
+  - [修改邮件模版支持 OTP](https://supabase.com/dashboard/project/xxxxxxxx/auth/templates)
+    - Confrim sign up: 首次注册的模版
+    - Magic link: 第二次登录模版
+- OTP 暴力破解防护 TODO
+  - https://supabase.com/dashboard/project/wtspqckturkzhstyjabx/auth/protection
+  - https://supabase.com/docs/guides/auth/rate-limits
+  - https://supabase.com/dashboard/project/wtspqckturkzhstyjabx/auth/rate-limits
+  - Supabase Auth Hooks
+- 客户端伪造钓鱼攻击 TODO
+- 域名白名单 TODO
+- email 登录 redirect 域名约束
+- 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)